test-data/unit/deps-types.test

```diff
@@ -550,6 +637,18 @@ class I: pass
 <mod.I.__init__> -> m
 <mod.I> -> <m.P.x>, <m.P>, m, m.P
 
+[case testAliasDepsNamedTupleFunctional]
+from typing import NamedTuple
+from mod import I
+A = I
+P = NamedTuple('P', [('x', A)])
+[file mod.py]
+class I: pass
+[out]
+<m.A> -> m
```

Comment:
Does this actually test that the dependency is generated, or would the dependency generate for just `A = I`, without the named tuple definition?